galley/pkg/runtime/processing/dispatcher.go

```diff
@@ -54,17 +63,26 @@ func NewDispatcherBuilder() *DispatcherBuilder {
 }
 
 // Add a new handler for the given Collection
-func (d *DispatcherBuilder) Add(t resource.Collection, h Handler) {
+func (d *DispatcherBuilder) Add(t resource.Collection, h Handler) *DispatcherBuilder {
```

Comment:
Will `Add()` and `Build()` ever be called concurrently?

Reply:
no